File: CS/CustomFormats/MyForms/MyAppointmentForm.ascx.cs


using System;
using DevExpress.Web.ASPxScheduler;
using DevExpress.XtraScheduler;

public partial class Templates_MyAppointmentForm : System.Web.UI.UserControl {
	protected void Page_Load(object sender, EventArgs e) {

	}
	public override void DataBind() {
		base.DataBind();
		MyAppointmentFormTemplateContainer container = (MyAppointmentFormTemplateContainer)Parent;
		AppointmentRecurrenceForm1.Visible = container.ShouldShowRecurrence;
       
		//btnDelete.Enabled = !container.IsNewAppointment;
	}
}

File: VB/CustomFormats/MyForms/MyAppointmentForm.ascx.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Web.ASPxScheduler
Imports DevExpress.XtraScheduler

Partial Public Class Templates_MyAppointmentForm
	Inherits System.Web.UI.UserControl
	Protected Sub Page_Load(ByVal sender As Object, ByVal e As EventArgs)

	End Sub
	Public Overrides Overloads Sub DataBind()
		MyBase.DataBind()
		Dim container As MyAppointmentFormTemplateContainer = CType(Parent, MyAppointmentFormTemplateContainer)
		AppointmentRecurrenceForm1.Visible = container.ShouldShowRecurrence

		'btnDelete.Enabled = !container.IsNewAppointment;
	End Sub
End Class
